src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -152,7 +143,7 @@ const EnvironmentPicker = ({
     useEffect(() => {
         if (!isDialogHidden) {
             dispatch({
-                type: SET_FIRST_TIME_VISIBLE,
```

Comment:
![image](https://user-images.githubusercontent.com/57726991/186236444-9faf8574-aaf2-40b4-a8a5-e261129aa984.png)

Reply:
had commented and marked as "resolved" this on the original PR